src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,117 @@
+Title: Change an application's series pre-juju2.3
+
+#  Change an application's series pre-juju2.3
+
+## Overview
+
+When deploying an application with juju you can specify the series it
+will use, or use the default series for the model.  Prior to juju 2.3,
+the application would always use the series specified at deployment
+when adding units with no way to change it.
+
+With juju 2.3 or later it's possible let juju know you'd like to add
+units with a different series moving forward by running the command:
+`juju update-series <application> <series>`
+
+Here we'll cover the manual steps on how to do this pre-juju2.3.
+
+## Prerequisites
+
+1. Check that the current charm revision in use for the application supports the
```

Comment:
To make a proper numbered list in markdown, these should be indented (two spaces is usual) and always start with a "1."

Reply:
updated